### PR TITLE
Octez 12.0

### PR DESCRIPTION
**Pull request by pirbo**
This is a new major release of the software suit for the tezos blockchain. It is required to run the consensus algorithm currently under vote by the community: Tenderbake.

### CODE REVIEW

**Comment by pirbo** [comment]
I think I reach a point were I'm happy with the state of ithe CI :-)

File: packages/tezos-error-monad/tezos-error-monad.12.0/opam [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "dune" { >= "2.9" }
+  "tezos-stdlib" { = version }
+  "data-encoding" { >= "0.4" & < "0.5" }
+  "lwt-canceler" { >= "0.3" & < "0.4" }
+  "tezos-lwt-result-stdlib" { = version }
+  "alcotest" { with-test & >= "1.5.0" }
+]
```
Comment (kit-ty-kate) [inline packages/tezos-error-monad/tezos-error-monad.12.0/opam:15]:
```suggestion
]
conflicts: [
  "result" {< "1.5"}
]
```

**Comment by pirbo** [comment]
Thank you! Indeed, I miss those 2... Unlees I don't read carefully again, the only remaining ones now despite the "Cancelled" are
- a (new) problem with the arch base image, no ?
- problems on arm32 where the linker complains about a definition made twice but I would like to blame the base image again because "works on my machine"© :-)

**Comment by kit-ty-kate** [comment]
Thanks! Sorry for the wait.

**Pull request merged**